src/kpro_lib.erl

```diff
@@ -137,6 +137,7 @@ encode(varint, I) when is_integer(I) -> kpro_varint:encode(I);
 encode(unsigned_varint, I) when is_integer(I) -> kpro_varint:encode_unsigned(I);
 encode(nullable_string, ?null) -> <<-1:16/?INT>>;
```

Comment:
What about this line? For backwards compatibility?

Reply:
no. `nullable_string` was invented for this purpose.
some `string` fields have the ambiguities of `null` and `""`, it's hard to know when to use `null` and when to use `""`.